CHANGELOG.md

```diff
@@ -1,27 +1,28 @@
-# Change Log for AMD SMI Library
+# Changelog for AMD SMI Library
 
-Full documentation for amd_smi_lib is available at [https://rocm.docs.amd.com/](https://rocm.docs.amd.com/projects/amdsmi/en/latest/).
+Full documentation for amd_smi_lib is available at [https://rocm.docs.amd.com/projects/amdsmi](https://rocm.docs.amd.com/projects/amdsmi/en/latest/).
 
 ***All information listed below is for reference and subject to change.***
 
 ## amd_smi_lib for ROCm 6.1.2
 
-### Added
+### Additions
 
 - **Added Ring Hang event**  
 Added `AMDSMI_EVT_NOTIF_RING_HANG` to the possible events in the `amdsmi_evt_notification_type_t` enum.
 
 - **Added process isolation and clean shader APIs and CLI commands**  
+
 Added APIs CLI and APIs to address LeftoverLocals security issues. Allowing clearing the sram data and setting process isolation on a per GPU basis. New APIs:
   - `amdsmi_get_gpu_process_isolation()`
   - `amdsmi_set_gpu_process_isolation()`
   - `amdsmi_set_gpu_clear_sram_data()`
 
-- **Added `MIN_POWER` to output of `amd-smi static --limit`**  
+- **Added `MIN_POWER` to output of `amd-smi static --limit`**.
 This change was to help users to identify what range they can change the power cap of the GPU to. We added this to simplify why a device supports (or does not support) power capping (also known as overdrive). See `amd-smi set -g all --power-cap <value in W>` or `amd-smi reset -g all --power-cap`.
```

Comment:
Looks good, Peter. Minor tweak,

Recommended:

This change helps users identify the range to which they can change the power cap of the GPU. The change is added to simplify why a device supports (or does not support) power capping (also known as overdrive). See `amd-smi set -g all --power-cap <value in W>` or `amd-smi reset -g all --power-cap.`